olive/engine/engine.py

```diff
@@ -30,6 +28,8 @@
 from olive.systems.utils import create_new_system_with_cache
 
 if TYPE_CHECKING:
+    from olive.engine.packaging.packaging_config import PackagingConfig
```

Comment:
Is there any linter can detect this?

Reply:
I could not find an easy way to detect these imports by linter